openstack/blockstorage/extensions/volumeactions/results.go

```diff
@@ -50,6 +55,101 @@ func (r commonResult) Extract() (map[string]interface{}, error) {
 	return s.ConnectionInfo, err
 }
 
+// VolumeImage contains information about volume upload to an image service.
```

Comment:
It represents the information stored in the `volume_type` object returned by the `volume_upload_image` result (An alternative name that is `VolumeUploadImage` rather than `VolumeImage`)

So it's `VolumeType` by nature, _but_ it's different than what would be stored under a `volumes/vx/types` package. So my suggestion of name was to make sure it's differentiated from that.

 I also suggested it because I didn't want someone to become confused by seeing `VolumeType` and wondering why it was only accessible through a `VolumeImage` result.

Third, I haven't scanned the rest of the volume actions to see if there isn't another action which would return yet another `VolumeType`, possibly using the same fields as `volumes/vx/types`. If I was confident that was not a possibility, I'd be fine with calling this `volumeactions.VolumeType` and handling questions about why two `VolumeType`s exist.

So, in the end, it's just trying to make sanity out of inconsistencies. :/

Thoughts / suggestions?